back-end/controllers/cloudinary.js

```diff
@@ -0,0 +1,27 @@
+const Cloudinary = (req, res) => {
+    const cloudinary = require("cloudinary"); //Cloudinary npm package
+    const image = req.body
+
+    console.log(image)
+    
+    cloudinary.config({ 
+        cloud_name: 'picme', 
+        api_key: process.env.CLOUDINARY_KEY, //ENV variables given to us through cloudinary, check google sheets
+        api_secret: process.env.CLOUDINARY_SECRET 
+    });  
+  
+    //Grabs from image link, will test grabbing image from fe later
+    cloudinary.uploader.upload("https://images.unsplash.com/photo-1534607287018-541c7d97748f?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=f8492e948d2ac5c91199a2623b1c42af&auto=format&fit=crop&w=925&q=80",

```

Comment:
I know that hard coded link is temporrary and just for testing but would still make the code a little clearer if you put that link in the image variable and then do cloudinary.uploader.upload(image).

Also, the image variable would be clearer if you called it imageUrl.